app/lib/irs1040_pdf.rb

```diff
@@ -1,92 +1,87 @@
 class Irs1040Pdf
-  attr_accessor :address
   include PdfHelper
 
   def source_pdf_name
     "f1040-TY2021"
   end
 
   def initialize(submission)
-    @submission = submission
-    @tax_return = submission.tax_return
-    @intake = submission.intake
-    @qualifying_dependents = submission.qualifying_dependents
-    @address = @submission.verified_address || @intake.address
-    @benefits = Efile::BenefitsEligibility.new(tax_return: @tax_return, dependents: @qualifying_dependents)
+    # For some PDF fields, use values from the database b/c the XML values are truncated or missing.
+    intake = submission.intake
+    @primary_first_name = intake.primary_middle_initial.present? ? "#{intake.primary_first_name} #{intake.primary_middle_initial}" : intake.primary_first_name
```

Comment:
it also feels like these make more sense inline where they're consumed rather than defining as instance variables.